image/helpers.go

```diff
@@ -142,11 +144,24 @@ func (tsto *ToolingStore) DownloadSnap(name string, revision snap.Revision, opts
 	baseName := filepath.Base(snap.MountFile())
 	targetFn = filepath.Join(targetDir, baseName)
 
-	pb := progress.NewTextProgress()
+	pb := progress.MakeProgressBar()
+	defer pb.Finished()
+
+	// Intercept sigint
+	c := make(chan os.Signal, 3)
+	signal.Notify(c, syscall.SIGINT)
+	go func() {
```

Comment:
Should we have the "signal.Reset(...)" from below as a defer here?

Reply:
if image is expected to have any use beyond the one-shot interactive use it currently has then the goroutine needs cleanup and the signal reset, indeed. Otherwise nah :-)